src/karmabot/karma.py

```diff
@@ -1,78 +1,44 @@
 import logging
 
-from karmabot.db import db_session
+import karmabot.bot as bot
+import karmabot.slack as slack
+from karmabot.db.database import database
```

Comment:
With my comment above you could import `Database` and instantiate it here rather.

Reply:
The problem is, that we need connect the database at startup but also allow the use of the instance in all other modules (e.g., commands). Otherwise, we have to create a new instance at every place where it is used. That was also the intention with the previous global var. 

Do you have an idea to do something like dependency injection?

Reply:
Yeah the "instance in all other modules" would be annoying, so let's leave this for now then. 
Yes "dependency injection" was the idea here, but you use monkeypatch for that right?